src/pylyrics/plot_cloud.py

```diff
@@ -0,0 +1,70 @@
+# Authors: Abhiket Gaurav, Artan Zandian, Macy Chan, Manju Abhinandana Kumar
+# January 2022
+
+from pylyrics import extract_lyrics
+from pylyrics import clean_text
+
+from wordcloud import WordCloud
+import matplotlib.pyplot as plt
+
+
+def plot_cloud(
+    song,
+    file_path,
+    max_font_size=30,
+    max_words=120,
+    background_color="black",
+    show=False,
+):
+    """
+    Creates a wordcloud of most occuring words in a string or list of strings
+
+    Parameters
+    ----------
+    song: python dictionary
+        with artist as dictionary key and song_title as value. Both key and value are strings.
+    file_path: str
+        The location to save the file
+    max_font_size: int, optional
+        maximum font size
+    max_words: int, optional
+        maximum number of words to be included in wordcloud
+    background_color: str, optional
+        background color
+    show: bool, default=False
+        whether to display the plot to screen
+
+    Returns
+    -------
+    image
+        A wordcloud image supported by matplotlib
+
+    Example
+    -------
+    >>> from pylyrics import plot_cloud
+    >>> plot_cloud(song,file_path,max_font_size, max_words, background_color, show)
+
+    """
+
+    ## Exception Handeling to be added
+
+    text = ""
+    # Create a string of all song lyrics
+    for artist, song_title in song:
+        raw_lyrics = extract_lyrics(artist, song_title)
```

Comment:
also need to add token in input

Reply:
FYI, you may want to check if raw_lyrics is None later, edge cases may break the function